src/PackageVersions/FallbackVersions.php

```diff
@@ -75,11 +83,14 @@ private static function getVersions(string $composerLockFile) : Generator
     {
         $lockData = json_decode(file_get_contents($composerLockFile), true);
```

Comment:
Naming (across this file overall) should be adjusted to identify that this is no longer `lockfile` data

Reply:
Now it will reflect the concept of "package data", since that's what we're after to begin with.